### DISCUSSION TITLE
Feature request: Tabs component

### DISCUSSION THREAD

**Discussion by itsMapleLeaf** [tailwindlabs/headlessui#429] [Ideas]
Examples from other libs:
- https://reach.tech/tabs
- https://reakit.io/docs/tab/

**Comment by livthomas** [comment]
I would also love to have such component in Tailwind UI. It could solve a lot of pain points such as horizontal scrolling on mobile devices.

**Comment by LeoniePhiline** [comment]
[vue3-accessible-tabs](https://github.com/jonbackus/vue3-accessible-tabs/) is another example, using Vue 3 and supporting [WAI ARIA keyboard navigation](https://www.w3.org/TR/wai-aria-practices/#tabpanel).

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

This is coming in the next couple weeks!
